sormas-ui/src/main/java/de/symeda/sormas/ui/UserProvider.java

```diff
@@ -70,6 +70,16 @@ public boolean hasUserRight(UserRight userRight) {
 		return getUserRights().contains(userRight);
 	}
 
+	public boolean hasUserRights(UserRight... userRights) {
```

Comment:
hasAllUserRights would be more specific.

You can use getUserRights().containsAll(...)